src/Data/Fin/Induction.agda

```diff
@@ -5,6 +5,7 @@
 ------------------------------------------------------------------------
 
 {-# OPTIONS --without-K --safe #-}
+{-# OPTIONS --warn=noUserWarning #-} -- for issue #1726
```

Comment:
It would be more useful to say  `for _≺_` because then we know when we can remove it.